test/edu/stanford/dlss/wowza/TestSulWowza.java

```diff
@@ -367,19 +373,24 @@ public void authorizeSession_validatesStacksToken()
     public void authorizeSession_getsUserIp()
     {
         IHTTPStreamerSession sessionMock = mock(IHTTPStreamerSession.class);
+        Map<String, String> mockHttpHeaderMap = new HashMap<String, String>();
+        mockHttpHeaderMap.put("x-forwarded-for", "127.6.6.6"); // userIp
+        when(sessionMock.getHTTPHeaderMap()).thenReturn(mockHttpHeaderMap);
         SulWowza spyModule = spy(testModule);
 
         spyModule.authorizeSession(sessionMock);
-        verify(sessionMock).getIpAddress();
+        verify(sessionMock).getHTTPHeaderMap();
```

Comment:
not sure if this is good enough as a test, but since the object is a Hash, is it worth doing a spy on a hash to ensure it looks for the right key?